templates.go

```diff
@@ -85,12 +76,14 @@ var templates = map[string][]string{
 		"github.com/u-root/u-root/cmds/uname",
 		"github.com/u-root/u-root/cmds/uniq",
 		"github.com/u-root/u-root/cmds/unshare",
-		"github.com/u-root/u-root/cmds/validate",
-		"github.com/u-root/u-root/cmds/vboot",
 		"github.com/u-root/u-root/cmds/wc",
 		"github.com/u-root/u-root/cmds/wget",
 		"github.com/u-root/u-root/cmds/which",
 	},
+	"boot": {
+		"github.com/u-root/u-root/cmds/*",
```

Comment:
I wouldn't include cmds/* in boot. boot should just be systemboot/*.

then you can do stuff like

```console
# for production
$ u-root -build=bb boot systemboot/uinit

# for console debugging
$ u-root -build=bb core boot
```